Update ESLint plugin v6 to v8

Hello,
I have noticed your ESLint plugin supports only ESLint version 6. Will be a nice update to version 8.
Thank you!

Just ran into this same problem

```
❯ npm i eslint-plugin-baseui --save-dev
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: my-app-cra-ts-redux-01@0.1.0
npm ERR! Found: eslint@8.11.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^8.11.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0" from eslint-plugin-baseui@10.10.0
npm ERR! node_modules/eslint-plugin-baseui
npm ERR!   dev eslint-plugin-baseui@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

Versions:
```
❯ npm --version
8.5.4
                                                                                                                                                                                                               
❯ node --version
v16.13.2
```

Packages
```
"eslint": "^8.11.0",
"baseui": "^10.10.0",
```

--- a/packages/eslint-plugin-baseui/package.json
+++ b/packages/eslint-plugin-baseui/package.json
@@ -9,7 +9,7 @@
   "license": "MIT",
   "author": "Graham Murdoch <murdoch@uber.com>",
   "peerDependencies": {
-    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0"
+    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0"
   },
   "devDependencies": {
     "babel-eslint": "^10.0.3",

